dockers/docker-database/Dockerfile.j2

```diff
@@ -32,7 +32,8 @@ RUN apt-get clean -y                                  && \
              s/^# unixsocket/unixsocket/;                \
              s/redis-server.sock/redis.sock/g;           \
              s/^client-output-buffer-limit pubsub [0-9]+mb [0-9]+mb [0-9]+/client-output-buffer-limit pubsub 0 0 0/; \
-             s/^notify-keyspace-events ""$/notify-keyspace-events AKE/ \
+             s/^notify-keyspace-events ""$/notify-keyspace-events AKE/; \
+             s/^databases [0-9]+$/databases 100/ \
```

Comment:
is this change expected? do we need 100 databases or it is changed for testing but later forget to remove

Reply:
The default value of databases is 16 which isn't enough for new databases of DPU.